command-expansion-server/.prettierrc

```diff
@@ -1,9 +1,11 @@
 {
-  "tabWidth": 2,
-  "parser": "typescript",
+  "trailingComma": "all",
   "singleQuote": true,
-  "semi": true,
   "arrowParens": "avoid",
+  "semi": true,
   "printWidth": 120,
-  "trailingComma": "all"
+  "tabWidth": 2,
+  "insertSpaces": true,
```

Comment:
Is `insertSpaces` a Prettier option? I can't find it [documented](https://prettier.io/docs/en/options.html). The default behavior of Prettier is `spaces` so I think it can be excluded.

Reply:
Oh, I must have found an older document as it was `editor.insertSpaces` but yeah the document you linked doesn't have it. I removed it.